src/alchemtest/amber/testfiles/descr.rst

```diff
@@ -0,0 +1,20 @@
+Amber: invalid/incomplete output files 
+======================================================
+
+Here we collected some invalid/incomplete amber output files that can be used to test specific part of the amber parser.
+
+Notes
+-----
+
+- no_atomic_section.out.tar.bz2: amberoutput file without the ATOMIC section
+- no_control_data.out.tar.bz2: amberoutput file without the '2.  CONTROL  DATA  FOR  ' section
+- no_dHdl_data_points.out.tar.bz2: amberoutput file with TI active, but no DV/DL values,
+- no_free_energy_info.out.tar.bz2: amberoutput file without the settings regarding the free energy calculation
+- no_results_section.out.tar.bz2: amberoutput file without the RESULT section
+- no_temp0_setted.out.tar.bz2: amberoutput file with temp0 not setted
+- no_useful_data.out.tar.bz2: amberoutput file without useful data, truncated after the header
+- none_in_mbar.out.tar.bz2: amberoutput file with a wrongly formatted MBAR section. Specifically, a lambda value in a MBAR section has been altered, so it dowsn't match with the other MBAR sections and the expected lambda values (0.2500 --> 0.2550)
+- not_finished_run.out.tar.bz2: amberoutput file from an unterminated run
+
+
+.. versionadded:: 1.0
```

Comment:
It is added in the next release, which we will make 0.7.0. If we *really* want to make a 1.0 (we don't have to be in sync with alchemlyb) then we can fix it before release.
```suggestion
.. versionadded:: 0.7.0
```

Reply:
However, I'd rather you don't duplicated invalidfiles/testfiles. Instead, renamed invalidfiles to testfiles and change the filenames in the `load_invalidfiles()`.

Then *this* becomes the DESCR for invalidfiles and you add a note
```suggestion
.. versionchanged:: 0.7.0
   Renamed `invalidfiles` directory to `testfiles`, renamed files to more expressive names,
   and added additional testcases for incorrectly formatted output files.
```